crates/bevy_internal/src/default_plugins.rs

```diff
@@ -21,6 +21,10 @@ plugin_group! {
         #[cfg(feature = "std")]
         #[custom(cfg(any(all(unix, not(target_os = "horizon")), windows)))]
         bevy_app:::TerminalCtrlCHandlerPlugin,
+        // NOTE: Load this before AssetPlugin to properly register http asset sources.
```

Comment:
This is an interesting case of plugin dependency ordering: usually, i would expect "A depends on B" to mean A must load *after* B, but here it is inverted, and we definitely dont want AssetPlugin to depend on HttpSourcePlugin. Something to have in mind for eventual plugin dependencies i guess.